app/src/main/java/io/appium/uiautomator2/core/AccessibilityNodeInfoDumper.java

```diff
@@ -95,9 +100,17 @@ private static String toXmlNodeName(@Nullable String className) {
         }
 
         String fixedName = className
+                .replaceAll("`", "")
                 .replaceAll("[$@#&]", ".")
                 .replaceAll("\\.+", ".")
                 .replaceAll("(^\\.|\\.$)", "");
+
+        if (((NormalizeTagNames) Settings.NORMALIZE_TAG_NAMES.getSetting()).getValue()) {
```

Comment:
Should we apply this replacement for espresso driver as well?

Reply:
I don't want to do any preliminary optimizations yet. The way class names are retrieved by espresso is different from UIA, so it might be there is no such problem.

Reply:
ah, I see > The way class names are retrieved by espresso is different from UIA

thanks!